tensorflow/python/ops/check_ops.py

```diff
@@ -87,6 +87,260 @@ def _shape_and_dtype_str(tensor):
+
+    ```python
+    with tf.control_dependencies([tf.debugging.{opname}(x, y)]):
+      output = tf.reduce_sum(x)
+    ```
+
+    {sym_name} means, for every element `x[i]` of `x`, we have `x[i] {sym}`.
+    If `x` is empty this is trivially satisfied.
+
+    Args:
+      x:  Numeric `Tensor`.
+      data:  The tensors to print out if the condition is False.  Defaults to
+        error message and first few entries of `x`.
+      summarize: Print this many entries of each tensor.
+      message: A string to prefix to the default message.
+      name: A name for this operation (optional).  Defaults to "{opname}".
+
+    Returns:
+      Op that raises `InvalidArgumentError` if `x {sym}` is False.
+      @compatibility(eager)
+        returns None
+      @end_compatibility
+
+    Raises:
+      InvalidArgumentError: if the check can be performed immediately and
+        `x {sym}` is False. The check can be performed immediately during 
+        eager execution or if `x` is statically known.
+    """.format(sym=sym, sym_name=cap_sym_name, opname=opname)
+    return func
+
+  return _decorator
+
+
+def _binary_assert_doc(sym):
+  """
+  Common docstring for most of the v1 assert_* ops that compare two tensors
+  element-wise.
+
+  Args:
+    sym: Binary operation symbol, i.e. "=="
+  """
+  def _decorator(func):
+    opname = func.__name__
+
+    func.__doc__ = """
+    Assert the condition `x {sym} y` holds element-wise.
+
+    This condition holds if for every pair of (possibly broadcast) elements
+    `x[i]`, `y[i]`, we have `x[i] {sym} y[i]`.
+    If both `x` and `y` are empty, this is trivially satisfied.
+
+    When running in graph mode, you should add a dependency on this operation
+    to ensure that it runs. Example of adding a dependency to an operation:
+
+    ```python
+    with tf.control_dependencies([tf.compat.v1.{opname}(x, y)]):
+      output = tf.reduce_sum(x)
+    ```
+
+    Args:
+      x:  Numeric `Tensor`.
+      y:  Numeric `Tensor`, same dtype as and broadcastable to `x`.
+      data:  The tensors to print out if the condition is False.  Defaults to
+        error message and first few entries of `x`, `y`.
+      summarize: Print this many entries of each tensor.
+      message: A string to prefix to the default message.
+      name: A name for this operation (optional).  Defaults to "{opname}".
+
+    Returns:
+      Op that raises `InvalidArgumentError` if `x {sym} y` is False.
+      @compatibility(eager)
+        returns None
+      @end_compatibility
+
+    Raises:
+      InvalidArgumentError: if the check can be performed immediately and
+        `x {sym} y` is False. The check can be performed immediately during 
+        eager execution or if `x` and `y` are statically known.
+    """.format(sym=sym, opname=opname)
+    return func
+
+  return _decorator
+
+
+def _make_assert_msg_data(sym, x, y, summarize, test_op):
+  """
+  Subroutine of _binary_assert that generates the components of the default
```

Comment:
Lint Error: Args section is missing, does not match arg list or is badly indented.
Expected: sym, x, y, summarize, test_op
Found: sym, summarize, test_op
Missing: x, y
Unexpected:

Reply:
Put `x` and `y` arguments onto separate lines of docstring instead of a single line.